### PR TITLE
Make blog link unique

### PR DESCRIPTION
**Pull request by 5archoufa**
Include date in URL

### CODE REVIEW

File: src/templates/Blog.jsx [hunk 1]
```diff
@@ -20,6 +20,9 @@ export default class Blog extends React.Component {
       post.category_id = config.postDefaultCategoryID;
     }
     const { date } = postNode.fields;
+
+    const codeBlockRegex = /```([\s\S]*?)```/g;
+
```
Comment (jdrueckert) [inline src/templates/Blog.jsx:25]: as you're not referencing `codeBlockRegex` anywhere, I assume this can be removed

**Review by jdrueckert — CHANGES_REQUESTED** [review]
From looking at the code changes (and ignoring the ones duplicating #239), this looks reasonable. I'll need to test it, though, before merging. I'll try to find time to do that this weekend, not sure I will though, so apologies as it might take another week before I get to merge this.

Meanwhile please remove the unused var declaration in `Blog.jsx` and the changes duplicating your other PR.

**Comment by jdrueckert** [comment]
@5archoufa I locally tested this and noticed two issues:

1. I don't think we need the entire date and timestamp in the blog link. Currently it's unlikely that we even write two blog posts in a month... Using the date should be enough even if the website picks up in blogs written again in the future.
2. The blog preview cards on the front page don't properly navigate to the correct post when clicking on "Read more". Seems like they still use the "old" (not unique) links.

Also, prettier complains about some formatting issues. You can check the warnings out locally using `yarn run format-check:js` and run prettier on the affected files using `yarn prettier <path/to/file> --write` to fix the formatting issues.

**Comment by Rukki13** [comment]
I've made a pull request here: #244

**Comment by Rukki13** [comment]
you can close the older one (#242)

**Comment by jdrueckert** [comment]
> I've made a pull request here: #244

Okay, then I'll close this one. But, just fyi, you don't need to open a new PR whenever you add changes. You can simply push to the same branch again and the PR should update.

**Pull request closed without merging**